node_freebsd.go

```diff
@@ -0,0 +1,24 @@
+package restic
+
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+func (node *Node) OpenForReading() (*os.File, error) {
+	file, err := os.OpenFile(node.path, os.O_RDONLY, 0)
+	if os.IsPermission(err) {
+		return os.OpenFile(node.path, os.O_RDONLY, 0)
```

Comment:
same comment here as in https://github.com/restic/restic/pull/168. This retry logic here is useless if it doesn't have NOATIME (if it failed the first time, it will just fail again the second time). Just simplify the function to only make the OpenFile call once.